.travis.yml

```diff
@@ -54,15 +54,14 @@ jobs:
       language: shell
 
     - name: "Deploy PyPI package"
-      install: echo "Skipping install stage"
+      install:
+        - pip3 install .
       script: echo "Skipping script stage"
       deploy:
         edge: true
         provider: pypi
         on:
           tags: true
-        distributions: sdist
```

Comment:
Removed these because afaics we should the travis defaults should be fine for `setuptools-scm`

Reply:
Hmm i see. Maybe its better to keep the `skip_existing` in case they change the default value of that